pallets/asset/src/lib.rs

```diff
@@ -1012,6 +1013,30 @@ decl_module! {
             };
             ClassicTickers::insert(&classic_ticker_import.ticker, classic_ticker);
         }
+
+        /// Forces transfers token from `portfolio_to` to pia default portfolio.
+        /// Only PIA is allowed to execute this.
+        ///
+        /// # Arguments
+        /// * `origin` Must be a PIA for a given ticker.
+        /// * `ticker` Ticker symbol of the asset.
+        /// * `value`  Amount of tokens need to force transfer.
+        /// * `portfolio_to` From whom portfolio tokens gets transferred.
```

Comment:
Let's call it `from_portfolio` / `from` / `source` / `source_portoflio` instead of `portfolio_to`.